### DISCUSSION TITLE
This game is blowing up.

### DISCUSSION THREAD

**Discussion by Bruce-Browninggg** [aeolus-1/triangleGame.github.io#60] [Ideas]
A random group of girls i saw was playing it. 9 people are currently playing it. Rohan found the infinity jump code and i think 3 people have said they are quitting. one minute later they kept playing. 

Also they like the tips, keep the dum fucking tips. but add the ones i said. Also they really want different levels and each level is a unique colour scheme rather than same level different colour. 

Main things people keep asking for is a menu screen and different levels, maybe you could but a map in the menu too.

**Comment by Bruce-Browninggg** [comment]
![Photo on 22-4-22 at 10 39](https://user-images.githubusercontent.com/104052498/164576820-389f0df2-ed21-4e18-88db-5a700c1fb6b6.jpg)

**Reply by Bruce-Browninggg** [reply]
everyone in the row infront and next to me is playing it...

**Reply by Quantaviousdingleberry420** [reply]
l + bozo + raitio + maidenless + git good + patty smeels like poo + my balls a sterio + my balls always itch and never not + touch grace + worst reef familt memeber

**Reply by serifonium** [reply]
frontite

**Reply by Quantaviousdingleberry420** [reply]
so true s͙͉̹̦̘͎̉̀̄͡ḙ̛̦͑͗́̂r̸̙͍̲̭͖̝̦̟ͥ͑̽ì͍̤̥́̂̇͜fͫ҉̦͙̝͈̤̼͎o̖͇͑̔͞n̶̩͚̱̣̱͚͛̿͒̃i̗̙̮̾̑ͪ́ͅų͔̮̺̜̩͕̺̈̄ͅm͙̼̪̦̣͚ͥ̿͐́ͅ

**Reply by Quantaviousdingleberry420** [reply]
<img width="549" alt="Screen Shot 2022-04-22 at 11 43 39 am" src="https://user-images.githubusercontent.com/104042540/164580092-009b6a00-5c0e-4c96-b1a7-3f26fba05dc1.png">
i hate seeing this on my screen

**Comment by aeolus-1** [comment]
i no care
